docs/pages/access-controls/guides/ip-pinning.mdx

```diff
@@ -15,8 +15,9 @@ This helps minimize the risk of compromised credentials being used from differen
 
 ## How it works
 <Admonition type="note">
-  **Observed IP** - IP of the client. Teleport records this from the direct connection of
-  the user or, if it's enabled, a load balancer through the PROXY protocol.
+  **Observed IP** - IP of the client. Teleport records this from the direct
+  connection of the user or, if it's enabled, a load balancer through the PROXY
```

Comment:
I would reword this. It's not clear that "it" refers to proxy protocol as written.

Maybe: Teleport records this from the direct connection of the user or via PROXY protocol or "X-Forwarded-For' headers if these features are enabled at the load balancer.